microsoft-edge/devtools-guide-chromium/contact.md

```diff
@@ -6,39 +6,40 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: devtools
-ms.date: 09/29/2021
+ms.date: 03/07/2022
 ---
 # Contact the Microsoft Edge DevTools team
 
-Send your feedback to help improve Microsoft Edge DevTools.  Contact the Microsoft Edge DevTools team to submit feature requests or bug reports of issues about developing for Microsoft Edge.
+Send your feedback to help improve Microsoft Edge DevTools.  Contact the Microsoft Edge DevTools team to submit feature requests or bug reports, or ask questions about Microsoft Edge DevTools.
 
-There are several ways to get in touch with the Microsoft Edge DevTools team.
+There are several ways to get in touch with the Microsoft Edge DevTools team:
+
+* Using the **Send feedback** window to quickly report bugs while using DevTools.
+* Using the **DevTools** repo to submit feature requests, questions, and have longer conversations with the team.
+* Using **The Web We Want** initiative to submit cross-browser suggestions.
+* Using the **edge-developer** repo to submit documentation issues.
 
 
 <!-- ====================================================================== -->
-## Use the Send Feedback window
+## Use the Send feedback window
 
-To report a bug in DevTools or to request a new feature, [open DevTools](index.md#open-devtools) and then select the **Send Feedback** icon.  Or, in DevTools, press `Alt`+`Shift`+`I` (Windows, Linux) or `Option`+`Shift`+`I` (macOS).  This approach creates a tracked issue for the Microsoft Edge development team.
+To report a bug or a feature request while you are using DevTools, click the **Send feedback** icon or press `Alt`+`Shift`+`I` (Windows, Linux) or `Option`+`Shift`+`I` (macOS).  This approach creates a tracked issue for the Microsoft Edge development team.
 
-:::image type="content" source="../media/devtools-send-feedback.msft.png" alt-text="The Send Feedback icon in Microsoft Edge DevTools." lightbox="../media/devtools-send-feedback.msft.png":::
+![Screenshot of Microsoft Edge, with DevTools open, and the Send feedback icon in DevTools' toolbar](media/devtools-send-feedback.png)
 
-Selecting the **Send Feedback** icon opens a **Send Feedback** window.
+Clicking the **Send feedback** icon opens a **Send feedback** window which you can use to describe the bug you found or the feature request you have.
```

Comment:
To open a window where you describe the bug or feature request, click **Send feedback**.

Reply:
The paragraph before the image already says to click the send feedback button. So this feels a bit like a repetition. I know how to fix this: I can re-structure this as a bullet list (first click button, then enter info in window), and then remove the first image, and only have one that shows both the button to click and the feedback window to use.